### PR TITLE
endpoint finished for all params for sample data

### PR DESCRIPTION
**Pull request by nightlaro**

### CODE REVIEW

File: server/index.js [hunk 1]
```diff
@@ -1,12 +1,68 @@
...
 app.use(express.static('public'));
+app.use(express.json());
+app.use(express.urlencoded())
 
 //console.log(__dirname + '/public/lib')
 
-var port = 3000;
+app.get('/:productId', (req, res) => {
+  var product_id = parseInt(req.params.productId);
+  var data = sampleData.find(data => data.product_id === product_id);
```
Comment (RaneWallin) [inline server/index.js:15]: Consider using destructuring to reduce the amount of typing needed later.

.e.g. 
`let { carousel_image, thumbnail, cover_image, description_image, description_gifs } = sampleData.find(data => data.product_id === product_id);`

That will eliminate needing to data `data.thumbnail`, `data.cover_image`, etc. and just use `thumbnail`, `cover_image`

File: server/index.js [hunk 2]
```diff
@@ -1,12 +1,68 @@
...
+  var type = req.query.type;
+
+  if (!type && product_id === data.product_id) {
+    res.send(data);
+  }
+
+  if (type === 'carousel_images') {
+    res.send({
+      product_id: data.product_id,
+      carousel_images: data.carousel_images
```
Comment (RaneWallin) [inline server/index.js:25]: If you do the destructuring thing above you can simplify this a lot. If the key and value are the same you only need to put it once. ie if you destructure out `product_id` then you could have `{ product_id: data.product_id }` which can be further simplified to just be `{ product_id }`.

File: server/index.js [hunk 3]
```diff
@@ -1,12 +1,68 @@
...
+/*
+Queries
+{ type: 'carousel_images' }
+Params
+{ productId: '123' }
+*/
+
+var port = process.env.PORT || 3000;
 app.listen(port);
 console.log(`Server Live listening on port ${port}`);
```
Comment (RaneWallin) [inline server/index.js:68]: Generally the console.log for the server is inside a callback function in the listen method()
``` javascript
app.listen(() => {
    console.log(`Listening on port ${port}`);
});
```
This way you only see the log if the server started successfully.

**Comment by nightlaro** [comment]
Implemented destructuring and placed console.log inside of app.listen callback

**Pull request merged**